at-luantran/Week3/2018_07_05_exercise4_rubyday3.rb

```diff
@@ -0,0 +1,53 @@
+class StringFormat
+  class << self
+  def human_name(string)
+    if string.is_a? String
+      i = 0
+      out = ''
+      string.strip!
+      string.gsub!(/[^A-Za-z ]/, '')
+      string.each_char do |x|
+        if string[i - 1] == ' ' && x != ' ' || i.zero?
+          _a = x.ord - 32
+          out += _a.chr
+        else
+          out += x unless(string[i - 1] == ' ' && x == ' ')
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/c08836bee3655e104837529ea8ff45ce098ad76f/at-luantran/Week3/2018_07_05_exercise4_rubyday3.rb#L14"></span>Layout/SpaceAroundKeyword: Space after keyword <code>unless</code> is missing.</td>
    </tr>
  </tbody>
</table>